pkg/auth/ldaputil/query.go

```diff
@@ -239,6 +260,9 @@ func QueryForEntries(clientConfig *LDAPClientConfig, query *ldap.SearchRequest)
 	glog.V(4).Infof("searching LDAP server %v://%v at dn=%q with scope %v for %s requesting %v", clientConfig.Scheme, clientConfig.Host, query.BaseDN, query.Scope, query.Filter, query.Attributes)
 	searchResult, err := connection.Search(query)
 	if err != nil {
+		if strings.Contains(err.Error(), "LDAP Result Code 32") {
```

Comment:
The error we get back from upstream isn't typed?  Also, does this happen?  We don't just get back an empty result set (which is what I would expect.)